lib/puppetserver/ca/certificate_authority.rb

```diff
@@ -1,4 +1,5 @@
 require 'json'
+require 'uri'
```

Comment:
I don't see why this is needed here?

Reply:
ah I was testing out where to put the query hash and forgot to remove that.  Thanks for the catch!